docs/man-pages/condor_version.rst

```diff
@@ -1,44 +1,75 @@
-      
-
 *condor_version*
-=================
+================
+
+Display current HTCondor installation details.
 
-print HTCondor version and platform information
-:index:`condor_version<single: condor_version; HTCondor commands>`\ :index:`condor_version command`
+:index:`condor_version<double: condor_version; HTCondor commands>`
 
 Synopsis
 --------
 
-**condor_version** [**-help** ]
+**condor_version** [**-help**]
 
-**condor_version** [**-arch** ] [**-opsys** ] [**-syscall** ]
+**condor_version** [**-arch**] [**-opsys**] [**-syscall**]
 
 Description
 -----------
 
-With no arguments, *condor_version* prints the currently installed
-HTCondor version number and platform information. The version number
-includes a build identification number, as well as the date built.
+Display the current HTCondor installation's version number and platform
+information.
 
 Options
 -------
 
  **-help**
-    Print usage information
+    Print usage information.
  **-arch**
-    Print this machine's ClassAd value for :ad-attr:`Arch`
+    Print this machine's computer architecture (:ad-attr:`Arch`).
  **-opsys**
-    Print this machine's ClassAd value for :ad-attr:`OpSys`
+    Print this machine's operating system (:ad-attr:`OpSys`).
  **-syscall**
-    Get any requested version and/or platform information from the
-    ``libcondorsyscall.a`` that this HTCondor pool is configured to use,
-    instead of using the values that are compiled into the tool itself.
-    This option may be used in combination with any other options to
-    modify where the information is coming from.
+    Get requested information from the ``libcondorsyscall.a``
+    that this HTCondor pool is configured to use instead of the
+    default built in values.
+
```

Comment:
Can we just remove the -syscall option in the code and in the docs?  It only made sense for standard universe which is long gone.